pkgs/data/icons/reversal-icon-theme/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, gtk3, gnome-themes-extra, gtk-engine-murrine, bc }:
+stdenv.mkDerivation rec {
+  pname = "reversal-icon-theme";
+  version = "unstable";
+
+  src = fetchFromGitHub {
+    repo = "Reversal-icon-theme";
+    owner = "yeyushengfan258";
+    rev = "fb6cb29e02991915931cf1ed9b88d5257f177ed2";
+    sha256 = "15vml083kd40hzfb6s1f5xndwqz6ys58sg076cvpz9vsbxhwbjnl";
+  };
+
+  buildInputs = [ gtk3 gnome-themes-extra gtk-engine-murrine bc ];
```

Comment:
I looked at some other themes and they listed as gtk3 as native input. I think this could also be the case here. Also is bc required for the installer or at runtime?

Reply:
Checked the installation script and noticed that `bc` was not a used package. Not certain about it being used during runtime since the author did not mention anything regarding that matter in their `README.md` and thus I decided to exclude it from the `default.nix` file.